app/modules/finance/operations_router.py

```
@@ -12,6 +12,10 @@ def _audit(db,p,a,e,i,s):
     from app.modules.scheduling.models import TtAuditEntry
     db.add(TtAuditEntry(school_id=p.school_id,actor=p.email or p.user_id,action=a,entity=e,entity_id=i,summary=s))
 def _money(v): return Decimal(str(v or 0))
+def _required_approvals(amount: Decimal) -> int:
+    if amount <= Decimal("50000"): return 1
```

Ruff (0.16.1)

[error] Multiple statements on one line (colon)

(E701)